design/Status/dc.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
+<path d="M2 9V11H22V9H2ZM2 13V15H7V13H2ZM9 13V15H15V13H9ZM17 13V15H22V13H17Z"/>
```

Comment:
dup of power_input

Reply:
I don't usually say this as I'm a fan of reuse, but I would strongly advise we bring in ac and dc on their own - separate from power_input and current (which I may recommend we change)

Reply:
I would say I agree. While they do exist in Material, they've got some goofy names. It's probably easier to point them to the PX Blue library of _electrical things_ than to create some sort of instructions that say 'oh, for AC power you use this goofy icon name from material and for DC use this other goofy name' where they're not even related to each other.

Reply:
This is specific to AC/DC...the other duplicates I would not include here.